src/main/java/com/robohorse/robopojogenerator/generator/PostProcessorFactory.java

```diff
@@ -16,6 +19,13 @@ public PostProcessorFactory() {
     }
 
     public AbsPostProcessor createPostProcessor(GenerationModel generationModel) {
+
+        // Not support AutoValue yet
+        if (generationModel.getLanguageItem().equals(LanguageItem.KOTLIN_DTO)
+                && !generationModel.getAnnotationItem().equals(AnnotationItem.AUTO_VALUE_GSON)){
+            return Injector.getAppComponent().newKotlinDataClassPostProcessor();
+        }
+
```

Comment:
i think to replace java/kotlin radio button with kotlin checkbox and add this flag into generation model. I think, that it will simplify logic